bin/rule.ml

```diff
@@ -266,7 +269,10 @@ let locks =
     Arg.(value & opt (list ~sep:',' string) [] & info [ "locks" ] ~doc ~docv)
 
 let cmd =
-  let doc = "Produce dune rules to synchronize markdown and OCaml files." in
+  let doc =
+    "Produce dune rules to synchronize markdown and OCaml files. mdx rule is \
```

Comment:
```suggestion
    "Produce dune rules to synchronize markdown and OCaml files. $(b, ocaml-mdx rule) is \
```

Reply:
I think it'll look cleaner like this. Also users might get confused since they invoke it as `ocaml-mdx` now!